recipes/crumble/meta.yaml

```diff
@@ -27,10 +27,15 @@ test:
 
 about:
   home: https://github.com/jkbonfield/crumble
-  #licence in describing paper is'OpenSource'. Asked for a specific type of license upstream https://github.com/jkbonfield/crumble/issues/7
-  license: open source
+  license: multiple BSD style licenses
+  license_family: BSD
+  #licence file is upstream in github, but not in release
+  #license_file: LICENSE.md
   summary: Controllable lossy compression of BAM/CRAM files
 
-extra: 
+identifiers:
```

Comment:
Please put this under `extra:`

Reply:
fixed. It might be a nice feature of linting to catch not wanted sections.

Reply:
Good idea

Reply:
@maarten-k Can you create an issue for that on bioconda-utils?